### DISCUSSION TITLE
VOYAGER relationships

### DISCUSSION THREAD

**Discussion by anthonyaniah** [thedevdojo/voyager#5457] [General]
When a relationship is created between two Models, only one field can be displayed. For instance:
surName, firstName, otherName columns cannot be selected to displayed all against on single row. Modification is needed to capture selection of multiple table fields on one row. Voyager is actually a powerful Admin Creator. Weldone, Team.

**Comment by fletch3555** [comment]
This can be done with an accessor method.  Here's the docs for [BREAD-action specific accessors](https://voyager-docs.devdojo.com/customization/bread-accessors), but the general concept is a [Laravel feature](https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor)
